pkgs/top-level/all-packages.nix

```diff
@@ -28861,6 +28861,17 @@ with pkgs;
 
   picom = callPackage ../applications/window-managers/picom {};
 
+  picom-next = picom.overrideAttrs (oldAttrs: rec {
+    version="vNext";
+      src = fetchFromGitHub {
+      owner = "yshui";
+      repo = "picom";
+      rev = "fPiLZ63+Bw5VCxVNqj9i5had2YLa+jFMMf85MYdqvHU=";
```

Comment:
This doesn't look like a valid commit.

Reply:
Updated (I accidentally pasted the wrong string.

Reply:
Checks: https://github.com/yshui/picom/commit/fade045eadf171d2c732820d6ebde7d1943a1397